auxx/src/Command/Proc.hs

```diff
@@ -321,32 +322,6 @@ createCommandProcs hasAuxxMode printAction mSendActions = rights . fix $ \comman
     , cpHelp = ""
     },
 
-    let name = "delegate-light" in
-    needsSendActions name >>= \sendActions ->
-    needsAuxxMode name >>= \Dict ->
-    return CommandProc
-    { cpName = name
-    , cpArgumentConsumer =
-      (,,,) <$> getArg tyInt "i"
-            <*> getArg tyPublicKey "pk"
-            <*> getArg tyEpochIndex "start"
-            <*> getArgOpt tyEpochIndex "end"
-    , cpExec = \(i, delegatePk, startEpoch, lastEpochM) -> do
-        CmdCtx{ccPeers} <- getCmdCtx
-        issuerSk <- (!! i) <$> getSecretKeysPlain
-        withSafeSigner issuerSk (pure emptyPassphrase) $ \case
-            Nothing -> logError "Invalid passphrase"
-            Just ss -> do
-                let psk = safeCreatePsk ss delegatePk (startEpoch, fromMaybe 1000 lastEpochM)
-                dataFlow
-                    "pskLight"
-                    (immediateConcurrentConversations sendActions ccPeers)
-                    (MsgTransaction OriginSender) psk
-                logInfo "Sent lightweight cert"
-        return ValueUnit
-    , cpHelp = ""
-    },
-
     let name = "delegate-heavy" in
```

Comment:
Perhaps rename `delegate-heavy` to simply `delegate` now?

Reply:
I decided to leave it because doing otherwise would break compatibility with some FAQs. And, anyway, light delegation will be _reworked_ afaiu so no need to call heavy delegation "just delegation".